src/reference/antora/modules/ROOT/pages/mqtt.adoc

```diff
@@ -548,3 +548,19 @@ public IntegrationFlow mqttOutFlow(
     return f -> f.handle(new Mqttv5PahoMessageHandler(clientManager));
 }
 ----
+
+NOTE: Starting with version 6.4, multiple instances of `MqttPahoMessageDrivenChannelAdapter` and `Mqttv5PahoMessageDrivenChannelAdapter` can now be added at runtime using corresponding `ClientManager` through `IntegrationFlowContext`
+
+[source,java]
+----
+private void addAddRuntimeAdapter(IntegrationFlowContext flowContext,Mqttv5ClientManager clientManager,
```

Comment:
Whitespace is supposed to be after each comma.
Also double check, please, that there are no tab indents in all those lines of code.